src/test/java/apprenticeship/HumanPlayerTest.java

```diff
@@ -0,0 +1,26 @@
+package apprenticeship;
+
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+import static org.junit.Assert.*;
```

Comment:
Do you really need to import all of `Assert`? Can you use `org.junit.Assert.<specific method>` for the methods that you call?